snapcraft/cli/_options.py

```diff
@@ -65,4 +65,4 @@ def get_project_options(**kwargs):
         target_deb_arch=kwargs.pop('target_arch'),
     )
 
-    return ProjectOptions(**project_args)
+    return ProjectOptions(**project_args, **kwargs)
```

Comment:
@kalikiana but in here you would be duplicating some arguments, with different names. 
There will be use_geoip and enable_geoip, parallel_builds and no_parallel_builds, target_deb_arch and target_arch.
This is terrible, it would be better if the arguments had the same names. But well, now that we are in this mess, wouldn't it be better to pop the debug argument from kwargs, and get it into project_args?